### PR TITLE
Live-125: Changed dateline colour neutral.20

### PR DESCRIPTION
**Pull request by dskamiotis**
## Why are you doing this?

Contrast of dateline colour needs to be clearer upon rendering to stand out on background.

## Changes

- Changed dateline colour to neutral.20
- Omitted text from import - deprecated 

## Screenshots

| Before | After |
| --- | --- |
|  |  |

### CODE REVIEW

File: src/components/dateline.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const darkStyles = darkMode`
 
 const styles = css`
     ${textSans.xsmall()}
-    color: ${text.supporting};
+    color: ${neutral[20]};
```
Comment (webb04) [inline src/components/dateline.tsx:26]: I think we just want this darker colour for opinion types and `text.supporting` for the others

File: src/components/shared/card.tsx [hunk 2]
```diff
@@ -208,6 +208,9 @@ const iconStyles = (format: Format): SerializedStyles => {
 }
 
 const icon = (itemType: RelatedItemType, format: Format): ReactElement | null => {
+    // console.log(itemType)
```
Comment (webb04) [inline src/components/shared/card.tsx:211]: Can remove these ㏒s

**Review by webb04 — CHANGES_REQUESTED** [review]

File: src/components/dateline.tsx (hunk 1 above)
Comment (dskamiotis) [reply src/components/dateline.tsx:26]: 👍 will change - also UI tests seem to be stalled

Comment (webb04) [reply src/components/dateline.tsx:26]: I think that's Chromatic picking up a UI change in the dateline component! can show you how to approve these once you've made the changes

**Comment by JamieB-gu** [comment]
I think the change for DCR will be to [this component](https://github.com/guardian/dotcom-rendering/blob/master/src/web/components/Dateline.tsx).

**Comment by webb04** [comment]
Think you'll have to update the stories to use the new `pillar` prop for `dateline` components https://github.com/guardian/apps-rendering/blob/bf62d6577909881f39abc60675bac51e5f03868e/src/components/dateline.stories.tsx

File: src/components/dateline.stories.tsx [hunk 3]
```diff
@@ -2,15 +2,18 @@
...
 
+type DefaultProps = {
+    pillar: Pillar
+}
 
 // ----- Stories ----- //
 
-const Default: FC = () =>
-    <Dateline date={some(new Date(date('Publish Date', new Date('2019-12-17T03:24:00'))))} />
+const Default: FC<DefaultProps> = ({pillar}:DefaultProps) =>
```
Comment (webb04) [inline src/components/dateline.stories.tsx:15]: I think you can pass pillar directly into Dateline like they pass the props in here? 

https://github.com/guardian/apps-rendering/blob/a35bbe2cc861099b157888a2692e2ecc8a81f665/src/components/anchor.stories.tsx#L19

Comment (dskamiotis) [reply src/components/dateline.stories.tsx:15]: Trying to replica the model with little success:
const Default: FC = () =>
    <Dateline 
        pillar={{
            pillar: Pillar
        }}

Comment (webb04) [reply src/components/dateline.stories.tsx:15]: I think what you have now is ok, you just might not need type and argument to the default component?

<Dateline pillar={pillar} date={some(new Date(date('Publish Date', new Date('2019-12-17T03:24:00'))))} />

Comment (dskamiotis) [reply src/components/dateline.stories.tsx:15]: coming up with error: Cannot find name pillar when i do that, thats why i initially used the prop DefaultProps to feed it in.

**Review by webb04 — APPROVED** [review]

**Pull request merged**